src/organisms/feedPageScollableContent/feedPageScrollableContent.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTheme } from 'styled-components';
+import { Footer } from '../../atoms/footer/Footer';
+import { Text } from '../../atoms/texts/text';
+import { NewUpdateSection, ScrollableContent, Wrapper } from './feedPageScrollableContent.styles';
+import { FeedPageScrollableContentProps } from './types';
+
+export const FeedPageScrollableContent = (
+  { children, newUpdates, onShowNew }: FeedPageScrollableContentProps,
+) => {
+  const theme: any = useTheme();
+  const newUpdateText = newUpdates && `Show ${newUpdates.length} new updates`;
+  return (
+    <Wrapper>
+      {newUpdates && (
+        <NewUpdateSection onClick={onShowNew}>
+          <Text color={theme.textShades.SHADE_MINUS_2} size="S-Regular">
+            {newUpdateText!}
```

Comment:
is this exclamation mark correct? 🤔

Reply:
@ArinCantCode  I thought it was safe to use `!` here as this Text component is wrapped in `newUpdates &&`, so it seems safe to assume that it will not be null/undefined?

Happy to implement another way if this is still not recommended though.